src/module/vue/sfmove-sheet.vue

```diff
@@ -25,7 +25,7 @@
         <hr class="nogrow" />
         <h4 class="flexrow nogrow">
           <span class="flexrow">Actions</span>
-          <icon-button icon="fa fa-plus" @click="addTrigger" />
+          <btn-faicon icon="plus block" @click="addTrigger" />
```

Comment:
Is this intentional? I mean, it ends up outputting `class="icon-button clickable fas fa-plus block"` but shouldn't this be something more like 

```suggestion
          <btn-faicon icon="plus" class="block" @click="addTrigger" />
```

…though the `btn-faicon` component doesn't have a class prop, so you'd need one. But it does seem like a step backwards semantically speaking to hide the "provide a CSS class" feature inside the `icon` prop.

Reply:
oops, guess i missed that one. it should have the attribute `class="block"`, yeah.

so far as i can tell, a class prop isn't required to make it inherit the classes applied to a component - it "just works" by appending them to any inherited classes.

example using a button in `legacy-track`:

<img width="427" alt="Screen Shot 2022-06-23 at 5 00 51 PM" src="https://user-images.githubusercontent.com/5354757/175434487-8918bee6-6162-48a0-ba07-db7326eca9c3.png">

<img width="344" alt="Screen Shot 2022-06-23 at 5 01 02 PM" src="https://user-images.githubusercontent.com/5354757/175434503-79e3aee4-9d9d-4692-af4d-f4c8e4c638c7.png">